components/configuration-mgt/org.wso2.carbon.identity.configuration.mgt.endpoint/src/gen/java/org/wso2/carbon/identity/configuration/mgt/endpoint/ResourceTypeApi.java

```diff
@@ -0,0 +1,148 @@
+ */
+
+package org.wso2.carbon.identity.configuration.mgt.endpoint;
+
+import io.swagger.annotations.ApiParam;
+import org.wso2.carbon.identity.configuration.mgt.endpoint.dto.ResourceTypeAddDTO;
+import org.wso2.carbon.identity.configuration.mgt.endpoint.dto.ResourceTypeDTO;
+import org.wso2.carbon.identity.configuration.mgt.endpoint.factories.ResourceTypeApiServiceFactory;
+
+import javax.ws.rs.Consumes;
+import javax.ws.rs.DELETE;
+import javax.ws.rs.GET;
+import javax.ws.rs.PATCH;
+import javax.ws.rs.POST;
+import javax.ws.rs.PUT;
+import javax.ws.rs.Path;
+import javax.ws.rs.PathParam;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.Response;
+
+@Path("/resource-type")
+@Consumes({"application/json"})
+@Produces({"application/json"})
+@io.swagger.annotations.Api(value = "/resource-type", description = "the resource-type API")
+public class ResourceTypeApi {
+
+    private final ResourceTypeApiService delegate = ResourceTypeApiServiceFactory.getResourceTypeApi();
+
+    @PATCH
+
+    @Consumes({"application/json"})
+    @Produces({"application/json"})
+    @io.swagger.annotations.ApiOperation(value = "Update existing resource type.\n", notes = "This API is used to update an existing resource type.\n", response = ResourceTypeDTO.class)
+    @io.swagger.annotations.ApiResponses(value = {
+            @io.swagger.annotations.ApiResponse(code = 200, message = "Ok"),
+
+            @io.swagger.annotations.ApiResponse(code = 400, message = "Bad Request"),
+
+            @io.swagger.annotations.ApiResponse(code = 401, message = "Unauthorized"),
+
+            @io.swagger.annotations.ApiResponse(code = 409, message = "Conflict"),
+
+            @io.swagger.annotations.ApiResponse(code = 500, message = "Server Error")})
+
+    public Response resourceTypePatch(@ApiParam(value = "This represents the resource type to update the existing.", required = true) ResourceTypeAddDTO type) {
+
+        return delegate.resourceTypePatch(type);
+    }
+
+    @POST
+
+    @Consumes({"application/json"})
+    @Produces({"application/json"})
+    @io.swagger.annotations.ApiOperation(value = "Create the resource type.\n", notes = "This API is used to create a new resource type.\n", response = ResourceTypeDTO.class)
+    @io.swagger.annotations.ApiResponses(value = {
+            @io.swagger.annotations.ApiResponse(code = 201, message = "Successful response"),
+
+            @io.swagger.annotations.ApiResponse(code = 400, message = "Bad Request"),
+
+            @io.swagger.annotations.ApiResponse(code = 401, message = "Unauthorized"),
+
+            @io.swagger.annotations.ApiResponse(code = 409, message = "Conflict"),
+
+            @io.swagger.annotations.ApiResponse(code = 500, message = "Server Error")})
+
+    public Response resourceTypePost(@ApiParam(value = "This represents the resource type to be added.", required = true) ResourceTypeAddDTO type) {
+
+        return delegate.resourceTypePost(type);
+    }
+
+    @PUT
+
```

Comment:
remove unnecessary new line

Reply:
Intentionally left due to a generated class.